src/core/utils/fileutils.cpp

```diff
@@ -60,7 +59,11 @@ bool FileUtils::copyRecursively( const QString &sourceFolder, const QString &des
   {
     QDir destDir( destFolder );
     if ( destDir.exists() )
-      destDir.removeRecursively();
+    {
+      bool success = destDir.removeRecursively();
+      if ( !success )
+        return false;
```

Comment:
If we assume this happens, let's add an explicit debug msg here, with reason if possible.

Reply:
Done